src/components/AboutUsPage/MeetTheTeam.js

```diff
@@ -9,39 +16,48 @@ const MeetTheTeam = () => {
 			role: "Founder",
 		},
 		{
-			image: zainPic,
-			name: "Zain Raza",
-			role: "Founder",
+			image: jagrajPic,
+			name: "Jagraj Aulakh",
+			role: "Developer Team",
+			link: "https://linktr.ee/juggy1233",
 		},
 		{
-			image: zainPic,
-			name: "Zain Raza",
-			role: "Founder",
+			image: hassanPic,
+			name: "Hassan Ahmad",
+			role: "Developer Team",
 		},
 		{
-			image: zainPic,
-			name: "Zain Raza",
-			role: "Founder",
+			image: jeremiePic,
+			name: "Jeremie Bornais",
+			role: "Consultant",
 		},
 		{
-			image: zainPic,
-			name: "Zain Raza",
-			role: "Founder",
+			image: wahidPic,
+			name: "Wahid Bawa",
+			role: "Cool Guy",
 		},
 		{
-			image: zainPic,
-			name: "Zain Raza",
-			role: "Founder",
+			image: cadenPic,
+			name: "Caden Quiring",
+			role: "Media Team",
 		},
 	];
 
 	return (
 		<PageSection className="text-center">
 			<h1 className="h1-text pb-16">MEET THE TEAM</h1>
-			<div className="flex flex-wrap justify-center">
+			<div className="max-w-screen-xl mx-auto flex flex-wrap justify-around">
 				{teamMembers.map((info, index) => (
-					<div
-						className="flex flex-col items-center px-10"
+					<a
+						href={
+							info.link
+								? info.link
+								: "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+						}
+						target="_blank"
+						className={`flex flex-col items-center w-40 p-4 my-2 mx-3
+							rounded-lg shadow-xl transition-color duration-200
+							hover:bg-gray-200`}
```

Comment:
would you wanna rename gray-200 so we can have as a set color

Reply:
does this matter? can't we rely on gray-200?

Reply:
would be easier to say something like hover-color or something, just makes it easy to read

Reply:
but it kinda is extra work to do